src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                            skos:prefLabel "large igneous province"@en .
 
 
 ###  http://sweetontology.net/realmGeol/LehmannDiscontinuity
 soreag:LehmannDiscontinuity rdf:type owl:Class ;
-                          rdfs:subClassOf soreag:GeologicBoundary ;
-                          rdfs:label "lehmann discontinuity"@en .
+                            rdfs:subClassOf soreag:GeologicBoundary ;
+                            rdfs:label "lehmann discontinuity"@en .
+
+
+###  http://sweetontology.net/realmGeol/LineageZone
+soreag:LineageZone rdf:type owl:Class ;
+                   rdfs:subClassOf soreag:BiostratigraphicUnit ;
+                   dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                   dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+                   skos:definition "The body of strata containing specimens representing a specific segment of an evolutionary lineage. (ICS, Online, December 2019)"@en ;
+                   skos:prefLabel "lineage zone"@en .
+
+
+###  http://sweetontology.net/realmGeol/Lithodeme
+soreag:Lithodeme rdf:type owl:Class ;
+                 rdfs:subClassOf soreag:LithodemicFeature ;
+                 dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                 dcterms:source <https://doi.org/10.1080/11035897.2016.1178666> ;
+                 skos:definition "A lithodeme is the fundamental formal unit in lithodemic classification and corresponds to formation in lithostratigraphy. A lithodeme may consist of a single rock type, several rock types (if they form a lithologically coherent unit) or several rock types that have become tectonically intermingled to form a lithological unit. A lithodeme may contain informal sub-lithodemes, some of which may be named phase or zone (e.g., mineralized zone, pegmatitic zone). The term complex can be used for a non-ranked lithodemic unit. (Kumpulainen, 2017)"@en ;
+                 skos:prefLabel "lithodeme"@en .
+
+
+###  http://sweetontology.net/realmGeol/LithodemicFeature
+soreag:LithodemicFeature rdf:type owl:Class ;
+                         rdfs:subClassOf soreag:GeologicFeature ;
+                         owl:disjointWith soreag:StratigraphicFeature ;
+                         dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                         dcterms:source <https://doi.org/10.1306/05230505015> ;
+                         skos:definition "A Lithodemic Unit is a defined body of predominantly intrusive, highly metamorphosed, or intensely deformed rock that, because it is intrusive or has lost primary structure through metamorphism or tectonism, generally does not conform to the Law of Superposition. Expressions of age of lithodemic units (because they are non-stratigraphic) should be given in terms of the appropriate geochronologic (geological-time) unit, and thus not by chronstratigraphic (time-rock) terms. (North American Stratigraphic Code, 2005)"@en ;
```

Comment:
LIthodeme is a lithodemic unit, so can't use in definition here

Lithodemic Feature is a GeologicUnit that does not conform to the Law of Superposition.

ScopeNote: Lithodemic Features typically consist of intrusive, metamorphosed, or deformed rock for which older and younger relations are not determined by overlying/underlying structural relations.  This is a parent class for the hierarchy of such units defined by the international Stratigraphic Code. 

yes-- there should be a **Geologic unit class**, following pattern in other geoscience ontologies e.g. GeoSciML 3.2, NADM